grapher/controls/entityPicker/EntityPicker.tsx

```diff
@@ -131,9 +135,15 @@ export class EntityPicker extends React.Component<{
         )
     }
 
+    private getColumn(slug: ColumnSlug | undefined): CoreColumn | undefined {
+        if (slug === undefined) return undefined
+        const col = this.manager.entityPickerTable?.getColumns([slug])[0]
```

Comment:
Could this instead be:

```suggestion
        const col = this.manager.entityPickerTable?.get(slug)
```

Reply:
Also, maybe instead of returning `undefined`, it could return the `MissingColumn` instance?

Columns have `.isMissing` method that can be used if needed.